packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -197,32 +235,55 @@ export const FeedContainer = ({
                 inputProps={{ onFocus: handleSearchFocus }}
                 suggestionsProps={suggestionsProps}
               />
-              {besideSearch}
-            </span>
+            </ConditionalWrapper>
           )}
-          {isV1Search && (
+          {isV1Search && !isMobile && (
             <span className="mt-4 flex flex-1 flex-row">
               <SearchBarSuggestionList
                 {...suggestionsProps}
-                className="mr-3 hidden tablet:flex"
+                className={classNames(
+                  'hidden tablet:flex',
+                  shouldUseFeedLayoutV1 ? 'mx-6 laptop:mx-0' : 'mr-3',
+                )}
               />
-              {actionButtons && (
+              {actionButtons && !shouldUseFeedLayoutV1 && (
                 <span className="ml-auto flex flex-row gap-3 border-l border-theme-divider-tertiary pl-3">
                   {actionButtons}
                 </span>
               )}
             </span>
           )}
-          <div
-            className={classNames(
-              'grid',
-              isV1Search && 'mt-8',
-              gapClass(isList, spaciness),
-              cardClass(isList, numCards),
+          {shouldUseFeedLayoutV1 && shortcuts}
+          <ConditionalWrapper
+            condition={shouldUseFeedLayoutV1}
+            wrapper={(child) => (
+              <div
+                className={classNames(
+                  'flex flex-col rounded-16 border border-theme-divider-tertiary laptop:mx-0 laptop:mt-6',
+                  isV1Search && 'mt-6',
+                )}
+              >
+                <span className="flex w-full flex-row items-center justify-between px-6 py-4">
+                  <strong className="typo-title3">
+                    {feedNameToHeading[feedName] ?? ''}
```

Comment:
This is only shown on v1 layout?

Reply:
Yes, the layout with the new header.

Reply:
Ah ok, just making sure it doesn't show on control :)